scss/_variables.scss

```diff
@@ -1312,18 +1312,18 @@ $dropdown-header-padding:           $spacer $spacer * .5 !default;
 // scss-docs-end dropdown-variables
 
 // scss-docs-start dropdown-dark-variables
-$dropdown-dark-color:               $gray-300 !default;
-$dropdown-dark-bg:                  $gray-800 !default;
-$dropdown-dark-border-color:        $dropdown-border-color !default;
-$dropdown-dark-divider-bg:          $dropdown-divider-bg !default;
+$dropdown-dark-color:               $white !default;
+$dropdown-dark-bg:                  $black !default;
+$dropdown-dark-border-color:        $gray-700 !default;
+$dropdown-dark-divider-bg:          $dropdown-dark-border-color !default;
 $dropdown-dark-box-shadow:          null !default;
 $dropdown-dark-link-color:          $dropdown-dark-color !default;
-$dropdown-dark-link-hover-color:    $white !default;
-$dropdown-dark-link-hover-bg:       rgba($white, .15) !default;
-$dropdown-dark-link-active-color:   $dropdown-link-active-color !default;
-$dropdown-dark-link-active-bg:      $dropdown-link-active-bg !default;
-$dropdown-dark-link-disabled-color: $gray-500 !default;
-$dropdown-dark-header-color:        $gray-500 !default;
+$dropdown-dark-link-hover-color:    $black !default;
+$dropdown-dark-link-hover-bg:       rgba($white, 1) !default;
```

Comment:
Why don't we use directly `$white` here?

Reply:
That's a good question, I just changed the value of that variable. I think it could be `$white`.